src/lib/server/db/session.ts

```diff
@@ -96,6 +100,31 @@ export async function invalidateSession(sessionId: string): Promise<void> {
 	}
 }
 
+// Invalidate sessions that exceed the maximum number of sessions
+export async function invalidateExcessSessions(userId: number): Promise<number> {
```

Comment:
Maybe put this select and delete within a transaction? @matijakljajic your opinion on this pls